### PR TITLE
Adds holy hand grenades.

### PR DESCRIPTION
**Pull request by RandomMarine**
Some assembly required.

Water and potassium makes explosions, but holy water and potassium didn't, until now!
Behaves identically to its non-blessed counterpart unless at least 150 units are mixed (Requiring large or better beakers). Where then blast size gets a slight boost (Still weaker than other explosive reagents) and the explosion is preceded with a righteous chorus. (Giving victims an extremely obvious warning to scramble)

### CODE REVIEW

**Comment by bgobandit** [comment]
someone's been playing bgo

**Comment by Incoming5643** [comment]
You don't need a new soundfile here, if you can believe it we already have https://github.com/tgstation/-tg-station/blob/master/sound/effects/pray.ogg

**Comment by RandomMarine** [comment]
I totally forgot that existed. Whoops.

**Comment by KorPhaeron** [comment]
Should make it do something to cultists/shadowlings etc in range. Maybe ignite them?

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm [hunk 1]
```diff
@@ -42,6 +43,20 @@
 	result_amount = 2
 	strengthdiv = 10
 
+/datum/chemical_reaction/reagent_explosion/potassium_explosion/holyboom
+	name = "Holy Explosion"
+	id = "holyboom"
+	required_reagents = list("holywater" = 1, "potassium" = 1)
+
+/datum/chemical_reaction/reagent_explosion/potassium_explosion/holyboom/on_reaction(datum/reagents/holder, created_volume)
+	if(created_volume >= 150)
```
Comment (KazeEspada) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: Greater than tho. Not exactly

**Comment by KazeEspada** [comment]
Also use praychaplain its longer

**Comment by RandomMarine** [comment]
Praychaplain is a little too long. I'll see about getting it to do something against revenants, cultists, and the like. Technically it would metaconfirm cultists, but it also punches holes in the station so it's not like you could toss 'em willy nilly without getting banned.

**Comment by lzimann** [comment]
@bgobandit or worms :^)

**Comment by Buggy123** [comment]
Interesting but until special anti-rev/cultist/etc effects are added it isn't really useful. Bluespace-beaker potassium-water grenades are already maxcap and the next size down, large beakers, is weak as hell.

Great for memes though, I guess. :+1:

**Comment by Iamgoofball** [comment]
@lzimann @bgobandit 
>>>>thinking Worms/BGO came up with the Holy Hand Grenade of Antioch

0/10 would not search for holy grails with

**Comment by Iamgoofball** [comment]
also this shouldn't be mixable unless in a Holy Hand grenade container, which would then cause the entire speech for throwing the grenade be stated by the thrower

**Comment by RandomMarine** [comment]
Revenants get stunned briefly (Long enough not to be able to escape the incoming blast) and are revealed for ten seconds upon witnessing the divine chorus.

Cultists, hatched shadowlings and their empowered thralls get stunned and ignited when within the blast (But the square-shaped check can also catch them outside, if there's an easy, round-shaped radius check, I'm not aware of it.)

**Comment by KazeEspada** [comment]
Make it mortalize an ascended shadowlings.

**Comment by YotaXP** [comment]
It should do nothing if you don't throw it exactly 3 seconds after priming it.

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm [hunk 2]
```diff
@@ -42,6 +43,33 @@
...
+		playsound(get_turf(holder.my_atom), 'sound/effects/pray.ogg', 80, 0, round(created_volume/48))
+		strengthdiv = 8
+		for(var/mob/living/simple_animal/revenant/R in get_hearers_in_view(7,get_turf(holder.my_atom)))
+			var/diety = ticker.Bible_deity_name
+			if(!ticker.Bible_deity_name)
+				diety = "Christ"
+			R << "<span class='userdanger'>The power of [diety] compels you!</span>"
+			R.stun(20)
+			R.reveal(100)
+		sleep(20)
```
Comment (KorPhaeron) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: Why the sleep?

Comment (Iamgoofball) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: probably for the sound to finish

Comment (RandomMarine) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: Like goof said, so the sound finishes before the boom.

Comment (KorPhaeron) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: will be a pretty terrible weapon if it has the grenade timer+2 second sleep. Easy to get out of the way of

Comment (YotaXP) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: Could still be good for throwing into a cult or alien hive.

Comment (duncathan) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: >not standing in awe of the holy hand grenade of antioch

smh tbh fam

**Comment by Iamgoofball** [comment]
instead of being a big blast on one epicenter

it should be a bunch of small blasts spread out across the shape of a cross

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm (hunk 2 above)
Comment (RandomMarine) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: That's the tradeoff for having a bigger boom. (200u yields 1,3,7 as opposed to 0,3,6)
It's not meant to be too practical, but something fun.

**Pull request merged**